### PR TITLE
Replaced modbus-tk with pymodbys library

### PR DESCRIPTION
**Pull request by Patfra**
Closes [issue](https://github.com/synchrotron-solaris/dev-solaris-adam/issues/1) from dev-solaris-adam
Eventually we decided to work with pymodbus.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,11 @@ RUN yum -y install epel-release \
     tango-db \
     tango-accesscontrol \
     tango-test \
+    python-pytango \
+    gcc \
```
Comment (LJBD) [inline Dockerfile:23]: Are you sure that `gcc-c++` doesn't have to be installed as well?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Supported tags and respective `Dockerfile` links
 
-* [`9`, `latest` (*Dockerfile*)](https://github.com/tango-controls/tango-cs-docker/blob/master/Dockerfile)
+* [`9`, `latest` (*Dockerfile*)](https://github.com/synchrotron-solaris/tango-cs-docker/blob/dev_libraries/Dockerfile)
```
Comment (LJBD) [inline README.md]: I believe there's no `9` tag in the DockerHub repository for this image.

**Review by LJBD — CHANGES_REQUESTED** [review]
The version tag in the README should be changed.

File: README.md (hunk 2 above)
Comment (Patfra) [reply README.md]: Oh I didn't saw that mistake.

File: Dockerfile (hunk 1 above)
Comment (Patfra) [reply Dockerfile:23]: `pip install pymodbys` didn't required that, so i thought  only `gcc` will be enough. The other thing is  that we want the image to be as small as possible.

Comment (LJBD) [reply Dockerfile:23]: Oh, okay, I thought that `pymodbus` doesn't require `gcc` (which is a C compiler), but it turns out that `twisted`, the networking library, needs it. Leave it as it is.

Comment (LJBD) [reply Dockerfile:23]: So remove `gcc-c++`, please.

Comment (Patfra) [reply Dockerfile:23]: Done.

Comment (LJBD) [reply Dockerfile:23]: Great! 👍

**Review by LJBD — APPROVED** [review]

**Pull request merged**